src/libs/SidebarUtils.ts

```diff
@@ -88,13 +88,16 @@ function getOrderedReportIDs(
             return;
         }
         const reportActions = allReportActions?.[`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${report.reportID}`] ?? {};
+        const parentReportAction = report.parentReportID && report.parentReportActionID
```

Comment:
Can we use `ReportActionsUtils.getParentReportAction` here?

Reply:
It's deprecated